roles/services/templates/systemd/layer-categories-lbg.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
```

Comment:
Instead of copy & paste I would use the templating engine that comes with ansible to make it parameterized. If you don't know how to do it, I can help.